doc/evaluation.rst

```diff
@@ -7,27 +7,27 @@ This section documents the exact mathematical definitions of the primary metrics
  
 The following conventions are used in the formulas in this section:
 
-:math:`N` - total number of responses in the :ref:`evaluation set<test_file>` with numeric human scores and numeric system scores. Zero human scores are, by default, excluded from evaluations unless :ref:`exclude_zero_scores<exclude_zero_scores_rsmtool>` was set to ``false``.
+:math:`N` = total number of responses in the :ref:`evaluation set<test_file>` with numeric human scores and numeric system scores. Zero human scores are, by default, excluded from evaluations unless :ref:`exclude_zero_scores<exclude_zero_scores_rsmtool>` was set to ``false``.
```

Comment:
Nitpick but isn't dash more common in these contexts?

Reply:
We can use a dash if you'd prefer but I don't like using dashes since they can look like minuses in a mathematical context which can be confusing.

Reply:
We can also use colons, FWIW.